### ISSUE TITLE
Realm.io "objects are the new api"

### ISSUE THREAD

**Issue by jbshirk** [ipfs/notes#200]
Hi folks, I just wanted to share this, I think worth knowing about, and of great interest to probably everyone here, definitely for core API, jsipfs, orbit, bigchain...
https://realm.io/news/serverless-alexander-stigsen-in-an-offline-first-world/

I've been lurking around here for maybe two years, watching, learning what I can. I'm subscribed to most of the repos. I'm very impressed with the pm development. I also see the struggles with the api. Here CRDTs replace all the complicated and confusing REST API stuff.

HTH

**Comment by diasdavid** [comment]
Hi @jbshirk! Did you find https://blog.ipfs.io/30-js-ipfs-crdts.md by chance? :)

**Comment by jon-choi** [comment]
Replacing deadlink: https://academy.realm.io/posts/serverless-alexander-stigsen-in-an-offline-first-world/

**Comment by jbshirk** [comment]
@diasdavid yes, I had maybe 3 years ago, and it looks much nicer these days! TY
